src/config.rs

```diff
@@ -59,47 +60,12 @@ pub struct Config {
     pub webapp: WebappConfig,
 }
 
-impl Default for Config {
-    fn default() -> Self {
-        use actix_web::http::header;
-        Self {
-            api: ApiConfig {
-                host: "api.noclick.me".to_string(),
-            },
-            cors: CorsConfig {
-                allowed_origins: vec![
-                    "https://noclick.me",
-                    "https://web.noclick.me",
-                    "https://www.noclick.me",
-                    "https://test.noclick.me",
-                    "https://app.noclick.me",
-                    "https://test.app.noclick.me",
-                ]
-                .iter()
-                .map(|s| s.to_string())
-                .collect(),
-                allowed_methods: vec![Method::GET, Method::POST],
-                allowed_headers: vec![header::ACCEPT, header::CONTENT_TYPE],
-            },
-            db: DbConfig {
-                // mode=read-write-create if it doesn't exist
-                url: "sqlite://db.sqlite?mode=rwc".to_string(),
-            },
-            limits: LimitsConfig {
-                global_requests_per_day: 10_000,
-            },
-            link: LinkConfig {
-                base_url: "https://noclick.me".to_string(),
-                max_length: 4096,
-            },
-            viewer: ViewerConfig {
-                host: "noclick.me".to_string(),
-            },
-            webapp: WebappConfig {
-                redirect_from_path: "/".to_string(),
-                redirect_from_host: "noclick.me".to_string(),
-                redirect_to: "https://app.noclick.me/".to_string(),
-            },
-        }
```

Comment:
I think it would make sense to provide sensible defaults, and let the config file only override what users want different from the defaults.

Reply:
Ok. Defaults for all the configuration? 

Also, do you want to make application run with defaults even if config file is not available?

Reply:
No, there are values that can't really have a sensible default, like the domain to use. A config file should be required with at least those values. But having to configure stuff like the daily limit should not be required. Specially for stuff we might add in the future, I wouldn't make the program fail to start because the config file is missing some value that can have a sensible default.

Reply:
I just had a look at the file and I guess anything that has the noclick.me host doesn't make to have as default.

Reply:
You mean configurations that has noclick.me do not require defaults?

Reply:
Yes, configuration for those should be mandatory as users need to own the used domain

Reply:
Done.